### PR TITLE
[ticket/13553] Message helper should accept a lang key var

### PR DESCRIPTION
**Pull request by VSEphpbb**
PHPBB3-13553

### CODE REVIEW

File: phpBB/phpbb/controller/helper.php [hunk 1]
```diff
@@ -197,15 +197,17 @@ public function error($message, $code = 500)
...
 	 * @param int $code The HTTP status code (e.g. 404, 500, 503, etc.)
 	 * @return Response A Response instance
 	 */
-	public function message($message, $title = false, $code = 200)
+	public function message($message, array $parameters = array(), $title = false, $code = 200)
 	{
+		array_unshift($parameters, $message);
 		$this->template->assign_vars(array(
-			'MESSAGE_TEXT'	=> $message,
+			'MESSAGE_TEXT'	=> call_user_func_array(array($this->user, 'lang'), $parameters),
```
Comment (nickvergessen) [inline phpBB/phpbb/controller/helper.php]: just call teh method directly as seen below?

Comment (VSEphpbb) [inline phpBB/phpbb/controller/helper.php]: lang doesn't accept the parameters as an array

Comment (nickvergessen) [inline phpBB/phpbb/controller/helper.php]: oh nevermind, got it

File: phpBB/phpbb/controller/helper.php [hunk 2]
```diff
@@ -197,16 +197,18 @@ public function error($message, $code = 500)
...
 	{
+		array_unshift($parameters, $message);
 		$this->template->assign_vars(array(
-			'MESSAGE_TEXT'	=> $message,
-			'MESSAGE_TITLE'	=> ($title === false) ? $this->user->lang('INFORMATION') : $title,
+			'MESSAGE_TEXT'	=> call_user_func_array(array($this->user, 'lang'), $parameters),
+			'MESSAGE_TITLE'	=> $this->user->lang($title),
 		));
 
 		return $this->render('message_body.html', $this->user->lang('INFORMATION'), $code);
```
Comment (VSEphpbb) [inline phpBB/phpbb/controller/helper.php]: @nickvergessen Should this be INFORMATION or $title?

**Comment by nickvergessen** [comment]
merged into prep via http://git.io/FHJY

**Pull request closed without merging**